backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -486,4 +476,109 @@ public static class FutureResult<V, E> {
     private V value;
     private E error;
   }
+
+  @AuthorizationConfiguration.RequirePermissionCSVUpload
+  public TestResultUpload processConditionAgnosticResultCSV(InputStream csvStream) {
+    var submissionId = UUID.randomUUID();
```

Comment:
a lot of this processing code is intentionally duplicated from the test result bulk upload. Plan is go back and refactor in the future once we have more understanding of how these different flows are / aren't similar